code/game/objects/items/weapons/implants/implant_misc.dm

```diff
@@ -48,6 +48,24 @@
 	imp_in.reagents.add_reagent("omnizine", 10)
 	imp_in.reagents.add_reagent("stimulants", 10)
 
+/obj/item/weapon/implant/adrenalin/deathsquad
+	name = "adrenal implant"
+	desc = "Removes all stuns and knockdowns."
+	icon_state = "adrenal"
+	origin_tech = "bluespace=4;biotech=10;combat=10;syndicate=4"
+	uses = 300
+
+/obj/item/weapon/implant/adrenalin/deathsquad/get_data()
+	var/dat = {"<b>Implant Specifications:</b><BR>
+				<b>Name:</b> Cybersun Industries Military Grade Adrenaline Implant<BR>
```

Comment:
Remember how the syndicate is secretive and that corporations aren't publicly affiliated with it.

It's actually kind of a hint about deathsquad's counter WHO I HAVE BEEN INFORMED WAS REMOVED FROM THE CODEBASE.